gui/launcher.lua

```diff
@@ -238,9 +244,14 @@ function EditPanel:init()
             key='CUSTOM_ALT_M',
             label=string.char(31)..string.char(30),
             on_activate=self.on_toggle_minimal},
+        widgets.Label{
+            view_id='pauselabel',
+            frame={l=13, t=3, w=21},
+            text={{key='D_PAUSE',key_sep=': ',text='no fort loaded',pen=COLOR_DARKGREY,key_pen=COLOR_GREEN}},
+            on_click=function() self:pause_fort() end},
         widgets.EditField{
             view_id='search',
-            frame={l=13, t=3, r=1},
+            frame={l=1, t=4, r=1},
```

Comment:
I'm not so sure about this change. @TaxiService what do you think?

![image](https://user-images.githubusercontent.com/977482/229654277-d3cb9c2d-15ac-4e07-85a0-cee5a2b41c00.png)

Reply:
like, moving those two things on a new line _does makes sense_ for readability, but...

...are there solutions that don't require adding a `Space: unpause` hotkeylabel at all?